File: 프로그래머스/lv0/120844. 배열 회전시키기/배열 회전시키기.py
def solution(numbers, direction):
    answer = [0]*len(numbers)
    if direction=='right':
        for i in range(len(numbers)):
            answer[i+1-len(numbers)]=numbers[i]
    else:
        for i in range(len(numbers)):
            if i==0:
                answer[-1]=numbers[0]
            else:
                answer[i-1-len(numbers)]=numbers[i]
            
    return answer

File: 프로그래머스/0/120844. 배열 회전시키기/배열 회전시키기.java
import java.util.*;
import java.util.stream.*;

class Solution {
    public ArrayList<Integer> solution(int[] numbers, String direction) {
        int len = numbers.length;
        ArrayList<Integer> answer = new ArrayList<Integer>();

        if(direction.equals("right")){
            answer.add(numbers[len-1]);
            answer.addAll(Arrays.stream(numbers, 0, len-1).boxed().collect(Collectors.toList()));
        } else {
            answer.addAll(Arrays.stream(numbers, 1, len).boxed().collect(Collectors.toList()));
            answer.add(numbers[0]);
        }

        return answer;
    }
}
